videocall-client/src/connection/connection.rs

```diff
@@ -53,13 +54,17 @@ impl Connection {
                 Callback::from(move |_| status.set(Status::Closed)),
             );
         }
+        let monitor = options.peer_monitor.clone();
         let mut connection = Self {
             task: Rc::new(Task::connect(webtransport, options)?),
             heartbeat: None,
+            heartbeat_monitor: None,
             status,
             aes,
         };
         connection.start_heartbeat(userid);
+        connection.start_heartbeat_monitor(monitor);
```

Comment:
up to you but how about setting `heartbeat_monitor` on 61 as opposed to calling `start_heartbeat_monitor` that way you save the method call to `start_heartbeat_monitor`

Reply:
yeah, that might be better way to do a one-liner, Thx :)